src/core/modules/SolarSystem.cpp

```diff
@@ -1686,7 +1687,7 @@ void SolarSystem::draw(StelCore* core)
 	}
 
 	// And sort them from the farthest to the closest. std::sort can split this into parallel threads!
-	std::sort(STD_EXECUTION_PAR_COMMA
+	std::sort(//STD_EXECUTION_PAR_COMMA
```

Comment:
Was this intended for this commit?

Reply:
Yes, temporarily, The auto-threaded sorting for some reason currently kills the profiler, and I think it's irrelevant for our discussion now. Error message a few sentences above.